test/unit/test_utils.py

```diff
@@ -18,6 +19,28 @@
 from ansible_runner.utils.streaming import stream_dir, unstream_dir
 
 
+def test_cleanup_folder(tmp_path):
+    folder_path = tmp_path / 'a_folder'
+    folder_path.mkdir()
+    assert folder_path.exists()  # sanity
```

Comment:
I think this assertion and the same on line 32 are testing `pathlib` or the state of the test environment, not our function, and are unnecessary.